### PR TITLE
Update Magento_Analytics ReadMe

### PR DESCRIPTION
**Pull request by diazwatson**
### Description (*)

Updates ReadMe for the Magento_Analytics module.

### Fixed Issues (if relevant)

1. magento/devdocs https://github.com/magento/devdocs/issues/4451: README content for modules - work in M2 Code

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/Analytics/README.md [hunk 1]
```diff
@@ -1,18 +1,28 @@
...
-* enabling subscription to the MBI and automatic re-subscription
-* changing the base URL with the same MBI account remained
-* declaring the configuration schemas for report data collection
-* collecting the Magento instance data as reports for the MBI
-* introducing API that provides the collected data
-* extending Magento configuration with the module parameters:
-    * subscription status (enabled/disabled)
-    * industry (a business area in which the instance website works)
-    * time of data collection (time of the day when the module collects data)
+- Enabling subscription to the MBI and automatic re-subscription
```
Comment (jeff-matthews) [inline app/code/Magento/Analytics/README.md]:
```suggestion
- Enabling subscription to Magento Business Intelligence (MBI) and automatic re-subscription
```

File: app/code/Magento/Analytics/README.md [hunk 2]
```diff
@@ -1,18 +1,28 @@
...
+- Changing the base URL with the same MBI account remained
```
Comment (jeff-matthews) [inline app/code/Magento/Analytics/README.md]: This is unclear. What do you mean?

File: app/code/Magento/Analytics/README.md [hunk 3]
```diff
@@ -1,18 +1,28 @@
...
+- Declaring the configuration schemas for report data collection
+- Collecting the Magento instance data as reports for the MBI
```
Comment (jeff-matthews) [inline app/code/Magento/Analytics/README.md]:
```suggestion
- Collecting the Magento instance data as reports for MBI
```

**Review by jeff-matthews — CHANGES_REQUESTED** [review]

File: app/code/Magento/Analytics/README.md (hunk 2 above)
Comment (diazwatson) [reply app/code/Magento/Analytics/README.md]: This was already there, it was a line introduced in https://github.com/magento/magento2/commit/bfc86089837665f9c61222f8fc5ee29ba3fe1262 by @olisenko

Do you know this user or how to contact him?
Maybe hw could explain hat this line means

Comment (jeff-matthews) [reply app/code/Magento/Analytics/README.md]: @olysenko, can you please help clarify what the this line means?

>  Changing the base URL with the same MBI account remained

Comment (sidolov) [reply app/code/Magento/Analytics/README.md]: @diazwatson , @jeff-matthews -  @olysenko our former employee, I'm not sure that we can receive the answer regarding this line. Probably, we may remove this line if it's confusing

Comment (jeff-matthews) [reply app/code/Magento/Analytics/README.md]: Thanks for the info @sidolov. We should definitely remove it if we can't clarify the meaning. I'm guessing it means that you can change the base URL of your Magento store without compromising the connection between the store and an existing MBI account that's been connected to the Magento app. Pinging @erikmarr (Data Solutions Technical Writer) to see if he can get us a quick answer on this one?

Comment (diazwatson) [reply app/code/Magento/Analytics/README.md]: Thanks @sidolov 👍 
Lets see what @olysenko says

Comment (sidolov) [reply app/code/Magento/Analytics/README.md]: @diazwatson , let's remove confusing lines and proceed with delivery

Comment (diazwatson) [reply app/code/Magento/Analytics/README.md]: Understood!

**Comment by diazwatson** [comment]
@sidolov requested changes were done

**Comment by ihor-sviziev** [comment]
Hi @jeff-matthews,
Could you review this PR?

**Review by jeff-matthews — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @jeff-matthews, thank you for the review.
[ENGCOM-5847](https://jira.corp.magento.com/browse/ENGCOM-5847) has been created to process this Pull Request

**Review by sidolov — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sidolov, thank you for the review.
[ENGCOM-5847](https://jira.corp.magento.com/browse/ENGCOM-5847) has been created to process this Pull Request

**Pull request merged**